src/containers/RecruiterDashboard/RecruiterJobPostDetails.jsx

```diff
@@ -0,0 +1,193 @@
+}));
+
+const JobPostDetails = () => {
+  const classes = useStyles();
+  //const params = useParams();
+  const params = "60bcffddc9068f6473ec271f";
+  const [openEmail, setOpenEmail] = useState(false);
+
+  const jobData = jobs.filter((post) => post._id.$oid === params)[0];
+  //console.log(jobData);
+  let candidates = jobApplicantsForPost.filter(
+    (el) => el.jobPostId === params
+  )[0].candidates;
+  console.log(candidates);
+  candidates = candidates.map((item) => {
+    //let candidateData = users.filter(user => user.jobseeker._id === item)
+    let currentStatus = candidateStatus.filter(
+      (el) => el._id.$oid === item.status
+    )[0].title;
+    let candidateData = users.filter(
+      (el) => el.jobseeker._id === el.candidate
+    )[0];
+    console.log(candidateData);
+    return { currentStatus };
+  });
+
+  console.log(candidates);
+  return (
+    <div className={classes.root}>
+      <Card>
+        <CardContent>
+          <Grid container justifyContent="left" spacing={2} direction="column">
+            <Grid item xs={12} md={12}>
+              <Typography variant="h3">{jobData.title}</Typography>
+              <Typography variant="subtitle2">
+                {recruiter.locations[0]}
+              </Typography>
+              <Typography variant="subtitle2">
+                Number of people applied:{" "}
+                <span className={classes.bold}>
+                  {jobData.applicants.length}
+                </span>
+              </Typography>
+            </Grid>
+            <Grid item xs={12} md={12}>
+              <Typography variant="h5" className={classes.title}>
+                Required Skills
+              </Typography>
+              <Grid container justifyCentent="left" spacing={2} direction="row">
+                {jobData.skills.split(",").map((skill, i) => (
+                  <Grid item key={i} className={classes.skill}>
+                    {skill}
+                  </Grid>
+                ))}
+              </Grid>
+            </Grid>
+            <Grid item xs={12} md={12} direction="row">
+              <Typography variant="subtitle2">
+                Locations:{" "}
+                <span className={classes.bold}>
+                  {" "}
+                  {jobData.locationsAt
+                    .map(
+                      (item) =>
+                        allCities.filter((city) => city._id.$oid === item)[0]
+                          .title
+                    )
+                    .join(", ")}
+                </span>
+              </Typography>
+            </Grid>
+            <Grid item xs={12} md={12}>
+              <Typography variant="h5" className={classes.title}>
+                Job Description
+              </Typography>
+              <Typography variant="body2">{jobData.description}</Typography>
+            </Grid>
+            <Grid item xs={12} md={12} justifyContent="center">
+              <SummaryComponent
+                cardTitle={"Ashwek Pawar"}
```

Comment:
Why hardcoded values? Make it dynamic

Reply:
i just had one candidate data and phot in recriter.js file for now.. yes but this card title will be having values from API and each summary component will be rendered through array.map()